src/main/java/org/profit/telegrambot/AntiHardcode.java

```diff
@@ -0,0 +1,64 @@
+package org.profit.telegrambot;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+
+import java.util.ArrayList;
+
+public class AntiHardcode {
+
+   static final String botDescription = """
+            This is a test bot for Java courses disguised as a tea shop.
+
+            It has two simple and one inline keyboards, can send images, and receive them.
+
+            More features coming soon :)""";
+
+   static final String greetingMessage = """
+            Welcome to our Tea Shop!
+
+            We have a wide selection of traditional Chinese tea for reasonable prices.
+
+            Just click 'Go Shopping' to see what we can offer you!
+
+            For any additional information click 'Shop's Info'""";
+
+    protected static final String token = "5793024716:AAHBn08nBwGe0D864bLUFX4UtFhLt412ppk";
+    static final String name = "R231Bot";
+    static int addCounter = 1;
+    static int pageCounter = 1;
+    static String imagePath;
+    static String photoCaption;
+    static final long totalItems = 6;
+    static ArrayList<Integer> products = new ArrayList<>();
+    static ReplyKeyboardMarkup keyboardMarkup = new ReplyKeyboardMarkup();
+
+    public static void chosePathAndDescription(int pageCounter) {
```

Comment:
IMHO: this function looks redundant here, but let's leave it here as for the first project. In further, do as much as possible to separate the logic and the resources. I mean, it is better to store resources with resources, and logic with logic. Do you guess why?)